Orange/widgets/unsupervised/owtsne.py

```diff
@@ -215,12 +226,32 @@ def stop(self):
     def resume(self):
         self.__set_update_loop(self.tsne_iterator)
 
+    def set_data(self, data: Table):
+        super().set_data(data)
+
+        if data is not None:
+            # PCA doesn't support normalization on sparse data, as this would
+            # require centering and normalizing the matrix
+            self.normalize_cbx.setDisabled(data.is_sparse())
+            if data.is_sparse():
+                self.normalize = False
+                self.normalize_cbx.setToolTip(
+                    "Data normalization is not supported on sparse matrices."
```

Comment:
Why is normalization not supported on sparse data?

Reply:
Normalization for sparse data is currently not supported anywhere where PCA is used. As far as I can tell, this includes the PCA, t-SNE and Louvain clustering widget.

I would prefer to add normalization support in a separate PR, adding it everywhere where it's missing, and leave this as is.